src/components/SendResetEmail.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { sendResetEmail} from "../redux/actions/resetPasswordAction";
+import { connect } from "react-redux";
+import cx from 'classnames';
+import { Link } from "react-router-dom";
+
+
+export class SendResetEmail extends React.Component {
+
+ state={
+   email:'',
+   isResetPasswordSent:false
+ }
+ componentWillReceiveProps({isResetPasswordSent}){
+    this.setState({
+      isResetPasswordSent
+    })
+  }
+
+  handleEmailChange=(e)=>{
+   this.setState(
+     {
+       email:e.target.value
+     }
+   )
+  }
+  handleSendEmailResetLink=()=>{
+    const {email}=this.state;
+    this.props.sendResetEmail(email);
+  }
+  render() {
+
+    return (
+      <div className="main-reset">
+        <div className="resetpasswordcontainer">
+          <div className="inner-reset-container">
+            <div className="txt-container">
+              <p className="p-reset-message">Forgot password</p>
+              <input
+                id="email"
+                type="email"
+                placeholder="Enter your email"
+                className="txt-input"
+                onChange={this.handleEmailChange}
+                value={this.state.email}
+              />
+              <div className="div-contaier">
+              <Link
+                  className={cx("btn","btn-cancel")}
```

Comment:
When I try to test LINK, Jest gives me an error.
If you are facing a similar issue I think you could alternatively use `window.location.asign('_pathname_')` or use [useHistory](https://reacttraining.com/react-router/web/api/Hooks/usehistory) is easier to test and I guess more efficient.